src/material-experimental/mdc-snack-bar/testing/snack-bar-harness.ts

```diff
@@ -36,11 +38,12 @@ export class MatSnackBarHarness extends ComponentHarness {
   }
 
   /**
-   * Gets the role of the snack-bar. The role of a snack-bar is determined based
-   * on the ARIA politeness specified in the snack-bar config.
+   * Gets the aria-live of the snack-bar's live region. The aria-live of a snack-bar is
+   * determined based on the ARIA politeness specified in the snack-bar config.
    */
-  async getRole(): Promise<'alert'|'status'|null> {
-    return (await this.host()).getAttribute('role') as Promise<'alert'|'status'|null>;
+  async getAriaLive(): Promise<AriaLivePoliteness> {
```

Comment:
This change would have to go through our regular deprecation process where the old method is marked as `@deprecated` for a major version before it's removed.

Reply:
Thanks for letting me know. I've restored the `getRole` method and marked it as `@deprecated` and a `@breaking-change` for `11.0.0`. Currently the method will always return `null`

Reply:
v11 is actually too soon; this one would have to be v13

Reply:
Updated to v13.